config/urls.py

```diff
@@ -2,45 +2,43 @@
 from __future__ import unicode_literals
 
 from django.conf import settings
-from django.conf.urls import include, url
+from django.conf.urls import url
 from django.conf.urls.static import static
 from django.contrib import admin
+from django.urls import include, path, re_path
 from django.views import defaults as default_views
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
 
-from rest_framework_swagger.views import get_swagger_view
-
+from simpl import __version__
+from simpl.games.apis.bulk_urls import bulk_router as bulk_api_router
 from simpl.games.apis.urls import router as api_router
 from simpl_users.apis.urls import router as simpl_users_api_router
 
-from simpl.games.apis.bulk_urls import bulk_router as bulk_api_router
-
-schema_view = get_swagger_view(title='Simpl Games API')
-
-urlpatterns = [
-    url(settings.ADMIN_URL, include(admin.site.urls)),
-]
-
-# User management
-urlpatterns += [
-    url(r'^accounts/', include('allauth.urls')),
-]
 
-# Static Media and User Uploads
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT, show_indexes=True)
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Simpl Games API",
+        default_version='v{}'.format(__version__),
+        description="Simpl Games Framework API Project",
+        # terms_of_service="https://www.google.com/policies/terms/",
+        # contact=openapi.Contact(email="contact@snippets.local"),
+        # license=openapi.License(name="GNU GENERAL PUBLIC LICENSE"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
 
 # Our application urls
-urlpatterns += [
-    url(r'^users/', include('simpl_users.urls', namespace='users')),
-
```

Comment:
Why did you remove the simpl_users.urls? Aren't they needed for django admin?

Reply:
This is part of the change you asked me to make and is dependent on: This is part of https://github.com/simplworld/simpl-users/pull/8

The Rest API endpoints are on #L36:

```python
path('apis/', include((simpl_users_api_router.urls, "simpl_users_api"))),
```